src/helpers/propOr.spec.js

```diff
@@ -0,0 +1,53 @@
+const test = require('tape')
+const helpers = require('../test/helpers')
+
+const bindFunc = helpers.bindFunc
+
+const isFunction = require('../core/isFunction')
+
+const propOr = require('./propOr')
+
+test('propOr function', t => {
+  const p = bindFunc(propOr)
+  const def = 'default value'
+
+  t.ok(isFunction(propOr), 'is a function')
+
+  const err = /propOr: String or integer required for second argument/
+  t.throws(p(def, undefined, {}), err, 'throws with undefined in second argument')
+  t.throws(p(def, null, {}), err, 'throws with null in second argument')
+  t.throws(p(def, false, {}), err, 'throws with false in second argument')
+  t.throws(p(def, true, {}), err, 'throws with true in second argument')
+  t.throws(p(def, 1.5, {}), err, 'throws with float in second argument')
```

Comment:
may want to throw in a falsey zero as well just to be safe.

Reply:
NOPE!! I am wrong. I should take care to look at signatures!
Sorry about that, this is perfect as is.